models/staking/solana/staking_solana_stake_actions.sql

```diff
@@ -0,0 +1,103 @@
+{{ config(
+    schema = 'staking_solana',
+    tags = ['dunesql'],
+    alias = alias('stake_actions'),
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time', 'unique_stake_action_id'],
+    post_hook='{{ expose_spells(\'["solana"]\',
+                                "sector",
+                                "staking",
+                                \'["ilemi"]\') }}')
+}}
+
+with    
+    merge as (
+        SELECT 
+            abs(aa.balance_change/pow(10,9)) as stake
+            , 'merge' as action
+            , account_sourceStakeAccount as source
+            , account_destinationStakeAccount as destination
+            , account_stakeAuthority as authority
+            , call_block_slot
+            , call_block_time
+            , call_outer_instruction_index
+            , call_inner_instruction_index
+            , call_tx_id
+        FROM {{ source('stake_program_solana', 'stake_call_Merge') }} m
+        LEFT JOIN {{ source('solana', 'account_activity') }} aa ON 1=1 
```

Comment:
we typically see poor performance on left joins to large tables

also, if it must remain left join due to logic constraints, then moving the incremental filter into the join condition should help (like we do on prices)